pyproject.toml

```diff
@@ -38,6 +38,7 @@ include = ['CHANGELOG.rst']
 python = '^3.6'
 django = '^3.0 ||^2.2'
 requests = '^1 || ^2'
+urllib3 = '>=1.26.0,<2.0'
```

Comment:
Hi, thank you for the PR and sorry for the dalay - I was on vacation.

Please change this into:
```suggestion
urllib3 = '^1.26.0'
```

![bilde](https://user-images.githubusercontent.com/5310116/144293352-4c4dc3bc-227e-49c2-b709-c76fedcfc32e.png)
[Docs.](https://python-poetry.org/docs/dependency-specification/#caret-requirements)

Reply:
done